powertools-logging/src/main/java/software/amazon/lambda/powertools/logging/LoggingUtils.java

```diff
@@ -93,6 +93,10 @@ public static void defaultObjectMapper(ObjectMapper objectMapper) {
     }
 
     public static ObjectMapper objectMapper() {
+        if(null == objectMapper) {
```

Comment:
Lazy load object mapper. We dont want to instantiate mapper if Logger is not configured to log payload